packages/mobile/src/account/saga.ts

```diff
@@ -44,19 +44,21 @@ export function* setPincode({ pincodeType }: SetPincodeAction) {
   }
 }
 
-function* clearStoredAccountSaga({ account }: ClearStoredAccountAction) {
+function* clearStoredAccountSaga({ account, softDelete }: ClearStoredAccountAction) {
   try {
-    yield call(removeAccountLocally, account)
-    yield call(clearStoredMnemonic)
-    yield call(ValoraAnalytics.reset)
-    yield call(deleteNodeData)
+    if (!softDelete) {
```

Comment:
nit:
```suggestion
    if (!softDelete) { return }
```

Reply:
We do want the last two lines with the `persistor.flush` and `restartApp` to run though.

Reply:
oh sorry didn't see that 👍